go.mod

```diff
@@ -32,7 +31,7 @@ require (
 	github.com/hashicorp/go-version v1.6.0
 	github.com/jpillora/backoff v1.0.0
 	github.com/json-iterator/go v1.1.12
-	github.com/kopia/kopia v0.10.7-0.20220330221649-08cf7eb936c6
+	github.com/kopia/kopia v0.12.2-0.20221217161912-f8be8f6a5698
```

Comment:
let's not include kopia upgrade in this PR. Can we raise another PR just for Kopia upgrade and in this PR we can just have the k8s deps upgrade?

Reply:
K8s dependency upgrades some otel packages which throws error while running `go mod tidy`. These packages are dependent on each other. Hence we need to upgrade both in same PR

Reply:
@ankitjain235 what do you think about this chaneg? PR looks good otherwise.

Reply:
Yes, this would be required.
We are just switching from https://github.com/kastenhq/kopia/commit/f1fc864f1dd09aa5103ead47f655ee302a5117ce which is some temporary commits above https://github.com/kopia/kopia/commit/f8be8f6a5698edba055291b138c117ec6eaae125 see https://github.com/kastenhq/kopia/commits/get-repo-upgrade-validate.
We are switching to https://github.com/kopia/kopia/commit/f8be8f6a5698edba055291b138c117ec6eaae125 which is effectively same.

Reply:
IIUC The kopia dependency is getting replaced by `kastenhq/kopia` in the `replace` directive above. Not sure why this change is required

Reply:
as discussed, even though we are not using upstream kopia version, this line updates kopia version and commit. this helps developers to figure out the details of upstream kopia we are using. hence not reverting the change